### ISSUE TITLE
Getting started documentation

### ISSUE THREAD

**Issue by nullr0ute** [fedora-iot/zezere#32]
Document in a README how to get started, DB options, configuration, authentication etc

**Comment by abitrolly** [comment]
Would be nice to get some tutorial as well. With info why and when it is needed and how to work with devices.

**Comment by septatrix** [comment]
Yes, I am currently considering using Fedora IoT for a project and have absolutely no clue what zezere is used for an what it is capable of: Is it only for pushing SSH keys? Can I initiate an update of the whole device fleet? ...

**Comment by nullr0ute** [comment]
> Yes, I am currently considering using Fedora IoT for a project and have absolutely no clue what zezere is used for an what it is capable of: Is it only for pushing SSH keys? Can I initiate an update of the whole device fleet? ...

We will be enhancing/refactoring zezere over the summer to use FIDO Device Onboarding protocol. It is not an entire device management platforms so is not used for updates, it's designed to be a component of a device management platform for onboarding devices

**Comment by gabbyprecious** [comment]
Hi @nullr0ute would love to work on this
